package.json

```diff
@@ -333,23 +334,25 @@
     "ts-loader": "6.2.2",
     "typesafe-actions": "2.2.0",
     "typescript": "4.4.4",
+    "typescript-plugin-css-modules": "^3.4.0",
```

Comment:
just confirming that these 2 should be in devDeps instead of dependencies

Reply:
well, sass-loader is used in the build system, so probably? Seems to be what some of these other ones are here for.

typescript-plugin-css-modules (sorry for forgetting that one from the security review) isn't even used in the build process, it's only picked up by language servers in IDEs like VSCode to provide intellisense for css classes.

Does that help @diracdeltas ?

Reply:
makes sense